lib/fog/openstack/requests/identity_v3/list_endpoints.rb

```diff
@@ -3,29 +3,22 @@ module Identity
     class OpenStack
       class V3
         class Real
-          def list_endpoints(options={})
-            params = Hash.new
-            params['service_id'] = options[:service_id] if options[:service_id]
-            params['interface'] = options[:interface] if options[:interface]
-
-            params['page'] = options.fetch(:page, 1)
-            params['per_page'] = options.fetch(:per_page, 30)
```

Comment:
@geemus we should not default values here, when they are already defaulted on keystone side, right? One source of defaults should be enough